docs/lcms/README.md

```diff
@@ -144,7 +144,7 @@ The specific type of assay being executed.
 
 | constraint | value |
 | --- | --- |
-| enum | `LC-MS (metabolomics)`, `LC-MS/MS (label-free proteomics)`, or `MS (shotgun lipidomics)` |
+| enum | `TMT LC-MS`, `Targeted Shotgun / Flow-injection LC-MS`, or `Untargeted LC-MS` |
```

Comment:
The list seems right, but I'm not sure how the old terms map to the new... but I don't think that detail matters here.

Reply:
I will check on this with Jeff Spraggins because we should know if these varieties of LC-MS are specific to the target molecule-types - 

eg. Is metabalomics always studied with LC-MS OR can metabolomics be studied with LC-MS/MS label-free or Untargeted LC-MS

Reply:
Ok: I've expanded dev-search to look at this: Currently, these mappings hold:

| ingest_validation_tools |   assay_types.yaml | count |
| --- | --- | --- |
| LC-MS (metabolomics) | Untargeted LC-MS | 32 |
| LC-MS/MS (label-free proteomics) | Untargeted LC-MS | 18 | 
| MS (shotgun lipidomics) | Targeted Shotgun / Flow-injection LC-MS | 8 |
| TMT (proteomics) | TMT LC-MS | 2 |

Conclusions:
- `TMT LC-MS` should be removed from this list
- ... and should be added to another list
- The bigger renaming this prompted can be handled latter: We want to map the strings as they currently they are before trying to make changes.

Reply:
(Never mind what I said about TMT LC-MS.)